cmd/internal/server/handlers/schema.go

```diff
@@ -15,5 +15,5 @@ func (Schema) Handle(ctx context.Context, psc *lib.PlanetScaleSource, db *lib.My
 	if err := (*db).BuildSchema(ctx, *psc, schemaBuilder); err != nil {
 		return nil, err
 	}
-	return schemaBuilder.(*fivetranSchemaBuilder).BuildResponse()
+	return schemaBuilder.(*FiveTranSchemaBuilder).BuildResponse()
```

Comment:
Had to make `FiveTranSchemaBuilder` public, because in server.go we refer to it when building our schema with metadata!